python/paddle/vision/datasets/voc2012.py

```diff
@@ -183,7 +205,7 @@ def __getitem__(self, idx):
 
         return data, label
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self.data)
 
     def __del__(self):
```

Comment:
```suggestion
    def __del__(self) -> None:
```

`__len__` 都标注了，这个一起标注一下吧